automatic/sqlite/update.ps1

```diff
@@ -58,7 +58,7 @@ function global:au_GetLatest {
         }
     }
 
-    @{ Version = $version; URL32 = $url32; URL64 = $url64; URLTools32 = $urlTools32; PackageName = 'SQLite' }
+    @{ Version = $version; URL32 = $url32; URL64 = $url64; URLTools64 = $urlTools64; PackageName = 'SQLite' }
```

Comment:
I know this code is just using what was there. But can we have these on separate lines rather than separating with semicolons? This would increase readability.

Reply:
Enhanced readability by avoiding single lines of code.